connectors/tests/test_utils.py

```diff
@@ -164,3 +168,72 @@ def test_get_base64_value():
     """This test verify get_base64_value method and convert encoded data into base64"""
     expected_result = get_base64_value("dummy".encode("utf-8"))
     assert expected_result == "ZHVtbXk="
+
+
+@pytest.mark.asyncio
+async def test_concurrent_runner(patch_logger):
+    results = []
+
+    def _cb(result):
```

Comment:
I find shortened function names like this really difficult to understand at a glance. And if tests aren't easy to understand, they serve much less use as developer documentation. I'd recommend naming the functions with full names like `_task_callback`. I still don't know what `coro` is supposed to be.

Reply:
`coro` is a pretty standard abbreviation for `coroutines` in Python (You will find this notation in the standard lib) but we can use `coroutine` for sure.